test/services/import/persist-licence.service.test.js

```diff
@@ -67,80 +72,97 @@ describe('Persist licence service', () => {
 
   describe('when given a valid transformed licence', () => {
     describe('and that licence does not already exist', () => {
-      it('creates a new licence record plus child records in WRLS and returns the licence ID', async () => {
-        const result = await PersistLicenceService.go(transformedLicence, transformedCompanies)
+      beforeEach(async () => {
+        // Call the thing under test
+        result = await PersistImportService.go(transformedLicence, transformedCompanies)
+
+        // Get the persisted data
+        newLicence = await _fetchPersistedLicence(transformedLicence.licenceRef)
+        company = await _fetchPersistedCompany(transformedCompany.externalId)
+        contact = await _fetchPersistedContact(transformedCompany.externalId)
+        address = await _fetchPersistedAddress(transformedCompany.addresses[0].externalId)
+      })
```

Comment:
Now this is placed here, and the test has been broken out into subtests, does this not mean we are first inserting, then repeatedly updating and fetching before each test?

Would this be better set as a `before()` block, so it only gets called once?